pom.xml

```diff
@@ -201,6 +201,11 @@
       <classifier>tests</classifier>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.googlecode.json-simple</groupId>
+      <artifactId>json-simple</artifactId>
+      <version>1.1.1</version>
+    </dependency>
```

Comment:
I don't like this direct dependencies.

JSONObject from jenkins already exist: https://github.com/jenkinsci/json-lib/blob/16052d1f74908fb59430eddd73f42aec2469b313/src/main/jdk15/net/sf/json/JSONObject.java#L125 with a toString method

The com.bettercloud.vault-java-driver already comes with a JSON parser.

You could also depend on https://github.com/jenkinsci/jackson2-api-plugin